contrib/slynk-stickers.lisp

```diff
@@ -242,7 +242,7 @@ after.")
                                       increment)
   "Return two values: a RECORDING and its position in *RECORDINGS*.
 Start searching from position FROM, an index in *RECORDINGS* which is
-successfuly increased by INCREMENT before using that to index
+successively increased by INCREMENT before using that to index
```

Comment:
Thanks, good catch. I think you can squash this commit in the other one where you fix the `loop` to be conforming.  If you're not comfortable enough with Git to do that, I can easily do it for you.

Reply:
I'm not comfortable enough with git to do this - can you?

Git and I have an uneasy relationship.

Reply:
Squashed.